task/06-conditions-n-loops-tasks.js

```diff
@@ -82,7 +110,10 @@ function getSumBetweenNumbers(n1, n2) {
  *   10,10,10 =>  true
  */
 function isTriangle(a,b,c) {
-    throw new Error('Not implemented');
+   if ((a + b) > c && (a + c) > b && (b + c) > a) {
```

Comment:
use `return <bolean expression>` instead of `if(<bolean expression>) return true else return false`